### PR TITLE
fix: Meta tags improvement #9

### PR DESCRIPTION
**Pull request by kubilaymelnikov**
added social media meta tags and more

### CODE REVIEW

File: lib/templates/mixins/seo/getMeta.js [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
     getMeta() {
-      const pageData = this.page.page
       const baseUrl = this.$typo3.options.baseURL
-      const meta = {
-        title:
-          pageData.meta.title ||
-          pageData.title ||
-          this.$nuxt.$options.head.title,
+      const { meta, socialMedia, title } = this.page.page
+      const { canonical, robots } = meta
```
Comment (mercs600) [inline lib/templates/mixins/seo/getMeta.js:6]: `robots` is available in TYPO3 headless ext >= 2.5. We should add fallback here to check if robots exist to keep backward compatibility. You can test it with https://api.t3pwa.com/ - now we have old version of headless ext here so `robots` field is not available.

**Comment by mercs600** [comment]
thank you @kubilaymelnikov for contribution, please check review comment.

File: lib/templates/mixins/seo/getMeta.js (hunk 1 above)
Comment (kubilaymelnikov) [reply lib/templates/mixins/seo/getMeta.js:6]: `robots` from `const { canonical, robots } = meta ` will result as undefined.

the simple solution would be to write `Object.keys(robots || {}) ` on line 36.

Comment (mercs600) [reply lib/templates/mixins/seo/getMeta.js:6]: I think it will be enough 👍

**Comment by mercs600** [comment]
Thank you @kubilaymelnikov I have merged it 🥳

**Pull request merged**